workflows/rnaseq/main.nf

```diff
@@ -155,7 +155,8 @@ workflow RNASEQ {
         params.umi_discard_read,
         params.stranded_threshold,
         params.unstranded_threshold,
-        params.skip_linting
+        params.skip_linting,
+        false
```

Comment:
no params for this?
Just false alsways?

Reply:
I don't really know why it's a module input, I don't think it's a core behaviour of FASTP. I'm struggling to see the use case for it to justify adding a pipeline param and wiring it in.

Reply:
Though I can if you really want...